modules/exploits/unix/http/schneider_electric_net55xx_encoder.rb

```diff
@@ -0,0 +1,188 @@
+﻿##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'net/ssh/command_stream'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::Udp
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::SSH
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Schneider Electric Pelco Endura NET55XX Encoder",
+      'Description'    => %q{
+        This module exploits an inadequate access control vulnerability creating a malicious
+        json request to the webUI encoder, thus allowing the SSH service to be enabled , and
+        changing the root password. This module has been tested sucessfully on: NET5501,
+        NET5501-I, NET5501-XT, NET5504, NET5500,NET5516,NET550 versions.
+
+
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Lucas Dinucci <idntk.lucdin@gmail.com>',
+          'Vitor Esperança <vitor@machiaveliclabs.com>'
+
+        ],
+      'References'     =>
+        [
+          ['CVE', '2019-6814'],
+          ['URL', 'https://www.schneider-electric.com/en/download/document/SEVD-2019-134-01/']
+        ],
+      'DefaultOptions'  =>
+        {
+          'EXITFUNC' => 'thread'
```

Comment:
I don't think this option applies to `ARCH_CMD` payloads.

Reply:
Resolved at commit [6f21abf](https://github.com/rapid7/metasploit-framework/pull/12084/commits/6f21abfe08bde97fbf2223f390b1e288c5da5721)